docs/reference/modules/node.asciidoc

```diff
@@ -49,6 +54,12 @@ document before indexing. With a heavy ingest load, it makes sense to use
 dedicated ingest nodes and to not include the `ingest` role from nodes that have
 the `master` or `data` roles.
 
+<<remote-node,Remote-eligible node>>::
+
+A node that has the `remote_cluster_client` role (default), which makes it
+eligible to act as a remote node. Any node in the cluster can act as a
+cross-cluster client and connect to remote clusters.
+
```

Comment:
I find this paragraph a bit confusing. I would say:
A node that has the `remote_cluster_client` role (default), which makes it eligible to act as a remote ~node~ **_client_**. **_By default,_** any node in the cluster can act as a cross-cluster client and connect to remote clusters.

Reply:
Thanks @fdartayre -- that clarification is helpful. I incorporated your suggestions in c2596c3.